napari/layers/_markers_layer/model.py

```diff
@@ -419,7 +419,7 @@ def _set_view_slice(self):
         self._sizes_view = sizes
 
         self._node.set_data(
-            data, size=sizes, edge_width=self.edge_width,
+            data[:, [1, 0]], size=sizes, edge_width=self.edge_width,
```

Comment:
My only complaint here is that this breaks my idea that coordinate transforms should happen in one place and one place only. Do you think that principle is workable in this PR?

Reply:
I fear this is unavoidable as vispy represents the order differently from how we do it - there will always need to be a flip in there somewhere. Soon we might be able to move the vispy `node` stuff to a separate view file, but we aren't there yet!

Reply:
I'm aware that there will always need to be a flip, but that flip should *always* look the same — a call to `data2vispy` or `vispy2data` or similar. Happy to postpone to a later PR, just wanted to flag it again so it stays near the top of our consciousness. Will merge now as I confirmed that it works correctly for me on all annotate/marker examples.